client/src/helpers/providers/__tests__/extractor.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { DataExtractor } from '../extractor'
+import * as mocks from './extractor.mocks'
+
+describe('DataExtractor', () => {
+
+  test('extractId', () => {
```

Comment:
les titres des describes et des tests devraient être plus verbeux: ex) test('should handle id extraction adequately ', ...)